### PR TITLE
DSG-2185: fix select bug

### PR DESCRIPTION
**Pull request by jgnuvalence**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

Ticket: https://dol-ewf.atlassian.net/browse/DSG-2185
fix bug that consists of the select not showing all the options when an item was selected

## Screenshot of the affected changes

image
image

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/ui/nuverial/src/lib/components/select/select.component.ts [hunk 1]
```diff
@@ -181,6 +181,7 @@ export class NuverialSelectComponent extends FormInputBaseDirective implements C
 
   private readonly _selectOptionsMap: Map<string, INuverialSelectOption> = new Map();
   private _selectOptions: INuverialSelectOption[] = [];
+  private _PrevieousValue = '';
```
Comment (derekdiep42) [inline libs/shared/ui/nuverial/src/lib/components/select/select.component.ts:184]: Typo, use `_previousValue`

Comment (jgnuvalence) [reply libs/shared/ui/nuverial/src/lib/components/select/select.component.ts]: thank you!, fixed

**Review by derekdiep42 — APPROVED** [review]
LGTM!

File: libs/shared/feature/form-nuv/src/lib/components/advanced/logic-validator/formly/formly-logic-validator.component.ts [hunk 2]
```diff
@@ -45,6 +47,15 @@ export class FormlyLogicValidatorComponent extends FormlyBaseComponent<LogicVali
...
       title: this.props.modalTitle,
     };
-    this._dialog.open(AlertModalComponent, dialogConfig).afterClosed().subscribe();
+
+    this._dialog
+      .open(AlertModalComponent, dialogConfig)
+      .afterClosed()
+      .pipe(
+        tap(_ => {
+          this._isModealOpen = false;
```
Comment (g8shen) [inline libs/shared/feature/form-nuv/src/lib/components/advanced/logic-validator/formly/formly-logic-validator.component.ts:56]: Is the spelling meant to be modeal?

**Review by g8shen — APPROVED** [review]

**Pull request merged**